lte/gateway/c/core/oai/tasks/amf/amf_as.cpp

```diff
@@ -1007,7 +1009,7 @@ static int amf_as_security_req(const amf_as_security_t* msg,
         } else {
           OAILOG_ERROR(LOG_AMF_APP, "UE not found : " AMF_UE_NGAP_ID_FMT "\n",
                        as_msg->ue_id);
-          return -2;
+          OAILOG_FUNC_RETURN(LOG_AMF_APP, -2);
```

Comment:
What's the reason for returning "-2" What action is taken with returning "-2".

Reply:
Changed it to RETURNerror.
Comment addressed.